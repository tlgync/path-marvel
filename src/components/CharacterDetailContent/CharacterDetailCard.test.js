import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { CharacterDetailContent } from '.';

test('CharacterDetailContent SnapShot', () => {
  const name = '3-D Manw';
  const path = 'http://i.annihil.us/u/prod/marvel/i/mg/c/e0/535fecbbb9784';
  const tree = renderer.create((
    <Router>
      <CharacterDetailContent name={name} path={path} />
    </Router>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
