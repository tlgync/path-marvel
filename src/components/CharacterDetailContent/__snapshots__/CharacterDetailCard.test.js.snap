// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterDetailContent SnapShot 1`] = `
<div
  className="detailContainer"
>
  <img
    alt="character"
    className="characterDetail-img"
    src="http://i.annihil.us/u/prod/marvel/i/mg/c/e0/535fecbbb9784.jpg"
  />
  <span
    className="characterDetail-name"
  >
    3-D Manw
  </span>
</div>
`;
